source/fundamentals/collations.txt

```diff
@@ -4,5 +4,311 @@ Collations
+``CreateCollection()`` or ``CreateView()`` method with your options object as an argument.
+
+.. _golang-create-collection:
+
+Create a Collection Example
+```````````````````````````
+
+The following example creates a new collection called ``books`` and specifies a default
+collation with the ``"fr"`` locale. The ``Strength`` collation field has a value of ``1``
+to ignore differences in letter accents.
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "fr", Strength: 1}
+   opts := options.CreateCollection().SetCollation(myCollation)
+   err := db.CreateCollection(context.TODO(), "books", opts)
+
+   if err != nil {
+      log.Fatal(err)
+   }
+
+Use the Default Collation Example
+`````````````````````````````````
+
+If you call an operation that supports collation on the ``books`` collection, the operation
+will now use the default collation.
+
+Assume the ``books`` collection contains the following documents:
+
+.. code-block:: json
+
+   {"name" : "Emma", "length" : "474"}
+   {"name" : "Les Misérables", "length": "1462"}
+   {"name" : "Infinite Jest", "length" : "1104"}
+   {"name" : "Cryptonomicon", "length" : "918"}
+   {"name" : "Ça", "length" : "1138"}
+
+.. note::
+
+   To learn how to insert documents, see :ref:`golang-insert-guide`.
+   
+You can use an operation with the default collation on the ``books`` collection specified
+in the :ref:`golang-create-collection`. The following example uses the ``Find()`` method
+to return all documents with a ``name`` value that alphabetically precedes ``"Infinite Jest"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"name", bson.D{{"$lt", "Infinite Jest"}}}}
+      cursor, err := coll.Find(context.TODO(), filter)
+
+      var results []bson.D
+      if err = cursor.All(context.TODO(), &results); err != nil {
+         panic(err)
+      }
+      for _, result := range results {
+         fmt.Println(result)
+      }
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{name Ça} {length 1138}]
+      [{name Cryptonomicon} {length 918}]
+      [{name Emma} {length 474}]
+
+Without specifying a default ``books`` collation, the ``Find()`` method would follow default
+binary collation rules to determine the ``name`` values that precede ``"Infinite Jest"``. These
+rules place words beginning with "Ç" after those beginning with "I". The output would resemble
+the following:
+
+.. code-block:: json
+   :copyable: false
+
+   [{name Cryptonomicon} {length 918}]
+   [{name Emma} {length 474}]
+
+See :ref:`golang-retrieve` for help with using the ``Find()`` method.
+
+.. _golang-index-collation:
+
+Set a Collation on an Index 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can apply a collation when you create a new index on a collection. The index stores
+an ordered representation of the documents in the collection so your MongoDB instance
+does not need to perform the ordering in-memory. 
```

Comment:
Suggestion: Could you clarify when the ordering in-memory takes place? I believe it would only be during a sort?

```suggestion
does not need to perform the ordering in-memory when you perform a sorting operation on your collection. 
```

Reply:
I clarified that this applies to sorting operations